2020/12/20 03:18:02 [emerg] 1#1: host not found in upstream "php-fpm:9000" in /etc/nginx/conf.d/default.conf:2
2020/12/20 03:28:53 [emerg] 1#1: host not found in upstream "php-fpm:9000" in /etc/nginx/conf.d/default.conf:2
2020/12/20 03:35:36 [error] 28#28: *1 "/var/www/public/index.php" is forbidden (13: Permission denied), client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2020/12/20 03:35:38 [error] 28#28: *1 "/var/www/public/index.php" is forbidden (13: Permission denied), client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2020/12/20 03:41:54 [error] 28#28: *1 "/var/www/public/index.php" is forbidden (13: Permission denied), client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2020/12/20 03:41:55 [error] 28#28: *1 "/var/www/public/index.php" is forbidden (13: Permission denied), client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2020/12/26 22:06:51 [warn] 28#28: *22 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/1/00/0000000001 while reading upstream, client: 172.18.0.1, server: localhost, request: "POST /dossier/extract HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "localhost", referrer: "http://localhost/dossier"
2020/12/26 22:07:21 [error] 28#28: *22 readv() failed (104: Connection reset by peer) while reading upstream, client: 172.18.0.1, server: localhost, request: "POST /dossier/extract HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "localhost", referrer: "http://localhost/dossier"
2020/12/27 19:01:57 [error] 28#28: *1 "/var/www/public/webgl/index.php" is forbidden (13: Permission denied), client: 172.18.0.1, server: localhost, request: "GET /webgl/ HTTP/1.1", host: "localhost"
2020/12/27 19:02:00 [error] 28#28: *3 open() "/var/www/public/webgl" failed (13: Permission denied), client: 172.18.0.1, server: localhost, request: "GET /webgl HTTP/1.1", host: "localhost"
2020/12/27 19:02:06 [error] 28#28: *3 open() "/var/www/public/webgl" failed (13: Permission denied), client: 172.18.0.1, server: localhost, request: "GET /webgl HTTP/1.1", host: "localhost"
2021/02/15 01:23:26 [warn] 29#29: *268 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/1/00/0000000001 while reading upstream, client: 172.18.0.1, server: localhost, request: "GET /dossier/creation?id_entite=2&id_formation=172 HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "localhost"
2021/02/15 01:23:57 [warn] 30#30: *270 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/2/00/0000000002 while reading upstream, client: 172.18.0.1, server: localhost, request: "GET /dossier/creation?id_entite=2&id_formation=172 HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "localhost"
2021/02/23 22:09:05 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /dossier/creation?id_entite=2&id_formation=172&coutht=12 HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "localhost", referrer: "http://localhost/propal/11/edit"
2021/12/08 14:06:23 [warn] 32#32: *85 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/1/00/0000000001 while reading upstream, client: 192.168.0.11, server: localhost, request: "GET /export/prospect HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "192.168.0.29", referrer: "http://192.168.0.29/contact"
2021/12/08 14:14:58 [warn] 32#32: *107 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/2/00/0000000002 while reading upstream, client: 192.168.0.11, server: localhost, request: "GET /export/prospect HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "192.168.0.29", referrer: "http://192.168.0.29/contact"
2021/12/08 14:15:16 [warn] 32#32: *107 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/3/00/0000000003 while reading upstream, client: 192.168.0.11, server: localhost, request: "GET /export/prospect HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "192.168.0.29", referrer: "http://192.168.0.29/contact"
2021/12/08 14:15:27 [warn] 32#32: *107 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/4/00/0000000004 while reading upstream, client: 192.168.0.11, server: localhost, request: "GET /export/prospect HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "192.168.0.29", referrer: "http://192.168.0.29/contact"
2021/12/08 14:46:45 [warn] 32#32: *119 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/5/00/0000000005 while reading upstream, client: 192.168.0.11, server: localhost, request: "GET /export/prospect HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "192.168.0.29", referrer: "http://192.168.0.29/contact"
2021/12/08 14:51:15 [warn] 32#32: *160 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/6/00/0000000006 while reading upstream, client: 192.168.0.11, server: localhost, request: "GET /export/a HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "192.168.0.29"
2021/12/08 15:12:55 [warn] 31#31: *2 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/1/00/0000000001 while reading upstream, client: 192.168.0.11, server: localhost, request: "GET /export/prospect HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "192.168.0.29", referrer: "http://192.168.0.29/contact"
2021/12/08 15:14:07 [warn] 31#31: *2 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/2/00/0000000002 while reading upstream, client: 192.168.0.11, server: localhost, request: "GET /export/prospect HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "192.168.0.29", referrer: "http://192.168.0.29/contact"
2021/12/08 15:14:49 [warn] 31#31: *2 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/3/00/0000000003 while reading upstream, client: 192.168.0.11, server: localhost, request: "GET /export/prospect HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "192.168.0.29", referrer: "http://192.168.0.29/contact"
2021/12/08 15:17:10 [warn] 31#31: *2 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/4/00/0000000004 while reading upstream, client: 192.168.0.11, server: localhost, request: "GET /export/prospect HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "192.168.0.29", referrer: "http://192.168.0.29/contact"
2022/01/03 15:18:24 [error] 34#34: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/login"
2022/01/03 15:18:26 [error] 34#34: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/login"
2022/01/03 15:18:26 [error] 34#34: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/login"
2022/01/05 12:55:46 [error] 32#32: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 12:55:47 [error] 32#32: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/01/05 12:56:44 [error] 32#32: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 12:56:44 [error] 32#32: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/01/05 12:56:44 [error] 32#32: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 12:56:44 [error] 32#32: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/01/05 12:56:45 [error] 32#32: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 12:56:45 [error] 32#32: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/01/05 12:56:52 [error] 32#32: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 12:56:52 [error] 32#32: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/01/05 12:57:45 [emerg] 41#41: bind() to 0.0.0.0:80 failed (98: Address in use)
2022/01/05 12:57:45 [emerg] 41#41: bind() to [::]:80 failed (98: Address in use)
2022/01/05 12:57:45 [emerg] 41#41: bind() to 0.0.0.0:80 failed (98: Address in use)
2022/01/05 12:57:45 [emerg] 41#41: bind() to [::]:80 failed (98: Address in use)
2022/01/05 12:57:45 [emerg] 41#41: bind() to 0.0.0.0:80 failed (98: Address in use)
2022/01/05 12:57:45 [emerg] 41#41: bind() to [::]:80 failed (98: Address in use)
2022/01/05 12:57:45 [emerg] 41#41: bind() to 0.0.0.0:80 failed (98: Address in use)
2022/01/05 12:57:45 [emerg] 41#41: bind() to [::]:80 failed (98: Address in use)
2022/01/05 12:57:45 [emerg] 41#41: bind() to 0.0.0.0:80 failed (98: Address in use)
2022/01/05 12:57:45 [emerg] 41#41: bind() to [::]:80 failed (98: Address in use)
2022/01/05 12:57:45 [emerg] 41#41: still could not bind()
2022/01/05 12:57:55 [error] 31#31: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 12:57:55 [error] 31#31: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/01/05 12:57:55 [error] 31#31: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 12:57:55 [error] 31#31: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/01/05 12:59:10 [error] 31#31: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 12:59:10 [error] 31#31: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/01/05 12:59:11 [error] 31#31: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 12:59:11 [error] 31#31: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/01/05 12:59:33 [notice] 45#45: signal process started
2022/01/05 12:59:37 [error] 46#46: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 12:59:37 [error] 46#46: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/01/05 12:59:37 [error] 46#46: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 12:59:37 [error] 46#46: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/01/05 13:01:49 [error] 32#32: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 13:01:50 [error] 32#32: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/01/05 13:01:51 [error] 32#32: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 13:01:51 [error] 32#32: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/01/05 13:01:57 [error] 32#32: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 13:01:57 [error] 32#32: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/01/05 13:05:43 [error] 33#33: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 13:05:43 [error] 33#33: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/01/05 13:08:37 [emerg] 1#1: "fastcgi_buffering" directive is not allowed here in /etc/nginx/nginx.conf:23
2022/01/05 13:11:16 [error] 34#34: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost"
2022/01/05 13:11:16 [error] 34#34: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/"
2022/02/01 08:40:37 [warn] 34#34: *2 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/1/00/0000000001 while reading upstream, client: 172.18.0.1, server: localhost, request: "GET /dossier/29170/convocation HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/dossier/29170/visualiser"
2022/02/01 08:42:51 [warn] 34#34: *4 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/2/00/0000000002 while reading upstream, client: 172.18.0.1, server: localhost, request: "GET /dossier/29170/convocation HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/dossier/29170/visualiser"
2022/02/01 08:45:07 [warn] 34#34: *27 an upstream response is buffered to a temporary file /var/cache/nginx/fastcgi_temp/3/00/0000000003 while reading upstream, client: 172.18.0.1, server: localhost, request: "GET /dossier/29170/convocation HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "localhost", referrer: "http://localhost/dossier/29170/visualiser"
